config/elasticsearch.yml

```diff
@@ -2,6 +2,8 @@ cluster:
   name: elasticsearch
 network:
   host: 0.0.0.0
+node:
+  name: ${HOSTNAME}
```

Comment:
Is the hostname always equals to k8s `metadata.name`?

Reply:
yes https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pods-hostname-and-subdomain-fields